docs/docs/channels/email/index.mdx

```diff
@@ -14,6 +14,21 @@ When creating an email provider integration you will be asked to provide additio
 
 For some email providers including SendGrid you will have to authenticate the **From email address** to make sure you will send email messages using an authorized address.
 
+## Sending Email Overrides
+
+The overrides field supports an email property that should have the interface of this:
+
+```ts
```

Comment:
Could we maybe provide the full novu.trigger snippet here for people to better understand the context?